boards/nucleo_h745zi_q_m7.json

```diff
@@ -35,9 +33,7 @@
     "protocol": "stlink",
     "protocols": [
       "jlink",
-      "stlink",
-      "blackmagic",
-      "mbed"
```

Comment:
Why you deleted these upload methods? They are still valid for this board.

Reply:
I can't test them. Thus they should stay away for the first time.

Reply:
Usually, that protocols work without problem, so feel free to revert them back.

Reply:
Reverted to support all suggested protocols.